tests/src/tests/string.spec.ts

```diff
@@ -113,16 +113,19 @@ export = () => {
 
 		const obj = {
 			[`str
-			ing`]: "foo"
+			ing`]: "foo",
 		};
 
+		// prettier-ignore
 		expect(obj[`str
```

Comment:
Just curious: what would prettier make this if not ignored?

Reply:
I don't actually think this `// prettier-ignore` is really necessary, I was just being cautious because it's hard to tell.

**Before**
```ts
		expect(obj[`str
			ing`]).to.equal("foo");
		expect(obj[key]).to.equal("foo");

		obj[`str
			ing`] = "bar";

		expect(obj[`str
			ing`]).to.equal("bar");
```

**After**
```ts
		expect(
			obj[
				`str
			ing`
			],
		).to.equal("foo");
		expect(obj[key]).to.equal("foo");

		obj[
			`str
			ing`
		] = "bar";

		expect(
			obj[
				`str
			ing`
			],
		).to.equal("bar");
```